src/publisher/fragment_logic.py

```diff
@@ -214,6 +214,21 @@ def delete_fragment_update_article(art, key):
         return set_all_article_json(art, quiet=False)
 
 #
+#
+#
+
+def location(av):
```

Comment:
not sure yet where this is used, but doesn't it seem dangerous to return strings as error values while the valid value is a string too? What about using a tuple or custom exceptions?